e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -102,6 +112,22 @@ def test_start_tensorboard_for_s3_experiment(tmp_path: Path, secrets: Dict[str,
             raise AssertionError(f"Did not find {SERVICE_READY} in output")
 
 
+@pytest.mark.slow  # type: ignore
+@pytest.mark.e2e_gpu  # type: ignore
+@pytest.mark.tensorflow2  # type: ignore
+def test_start_tensorboard_for_s3_experiment(tmp_path: Path, secrets: Dict[str, str]) -> None:
+    start_tensorboard_for_s3_experiment(tmp_path, secrets)
+
+
+@pytest.mark.slow  # type: ignore
+@pytest.mark.e2e_gpu  # type: ignore
+@pytest.mark.tensorflow2  # type: ignore
+def test_start_tensorboard_for_s3_prefix_experiment(
+    tmp_path: Path, secrets: Dict[str, str]
+) -> None:
+    start_tensorboard_for_s3_experiment(tmp_path, secrets, "my/test/prefix")
```

Comment:
you can keep the original function and use `@pytest.mark.parameterize(prefix, [None, "my/test/prefix])` do do this a little simpler.

Reply:
done